docs/reference/buildx_imagetools_inspect.md

```diff
@@ -45,7 +64,492 @@ Manifests:
 
 Same as [`buildx --builder`](buildx.md#builder).
 
+### <a name="format"></a> Format the output (--format)
+
+Format the output using the given Go template. Defaults to `{{.Manifest}}` if
+unset. Following fields are available:
+
+* `.Name`: provides the reference of the image
+* `.Manifest`: provides manifest or manifest list
+* `.Image`: provides the image config as `application/vnd.oci.image.config.v1+json` mediatype
```

Comment:
what does this "as mediatype" mean?

Reply:
Yes that's it I will remove that. Was taken from https://github.com/opencontainers/image-spec/blob/1f308fd27264a52f6a089202b1ff9dc15ff27398/specs-go/v1/config.go#L81